release-doc-in-process/all_vf.TTL

```diff
@@ -427,6 +441,47 @@ vf:resourceConformsTo
         rdfs:range      vf:ResourceSpecification ;
         rdfs:comment    "The primary resource knowledge specification or definition of an existing or potential resource." .
 
+vf:intents
```

Comment:
I think we should name property based on meaning of the relationship, not the type of the object. Also using plural stays consider as anti pattern. Any property can take a single value or an array (speaking bit JSON-LD specific way)
Some options
* *eligibleIntent*
* *applicableTo*

Reply:
removed this one anyhow - but I'll try! (seems there is a sweet spot between just repeating the class and something that has to be looked up to see what it is)